### PR TITLE
assets: Fix up the topology view icon colors and lines

### LINKED ISSUE #5952: Icons and connecting lines are not dark enough in network view
For the web console, the icons and the connecting lines are not dark enough making them hard to see. Particular standouts are the route icon, the deployment config, and the connecting lines. 

Please increase the contrast of these pieces with the diagram. 

screenshot from 2015-11-18 09-47-06

### PR DESCRIPTION
**Pull request by stefwalter**
* Make the lines darker in the topology view.
 * Use Patternfly font icons instead of SVGs.
 * Add color to the route icon.
 * Make DeploymentConfig icon color same as Deployment

Closes #5952

### CODE REVIEW

**Comment by stefwalter** [comment]
screenshot from 2015-12-09 14-31-42

File: assets/app/styles/_topology.less [hunk 1]
```diff
@@ -12,16 +12,25 @@ kubernetes-topology-graph {
...
+  }
+
   // These links are rendered dotted, where relationship incidental
   line.ServiceReplicationController,
   line.ServiceDeploymentConfig,
   line.DeploymentConfigReplicationController,
   line.ReplicationControllerPod {
-    stroke-dasharray: 1, 3;
+    stroke-dasharray: 5, 2;
     strake-linecap: round;
```
Comment (jwforres) [inline assets/app/styles/_topology.less]: is this a typo here? stroke-linecap?

Comment (stefwalter) [inline assets/app/styles/_topology.less]: Thanks. Fixed.

**Comment by jwforres** [comment]
ready to merge this?

**Comment by stefwalter** [comment]
> ready to merge this?

Yup. Should be good to go.

**Comment by jwforres** [comment]
[merge]

**Comment by jwforres** [comment]
[merge]

**Comment by jwforres** [comment]
[merge]

**Comment by jwforres** [comment]
[merge]

**Comment by jwforres** [comment]
[merge] i mean it this time

**Comment by jwforres** [comment]
[merge]

**Pull request merged**